cypress-io/cypress
# fix: Cleanup RunStatusDots component

- Closes #22629 , #22598 

### User facing changelog
* Styling fixes to "Latest Runs" tooltip to keep it centered over the latest run entry

### Additional details
* Logic cleanup - conditional that wraps a for loop wasn't being as restrictive as it could be

### Steps to test
1. Open a project with cloud recordings in the app, ensure you're logged in so that Latest Run data is loaded
2. In Specs Explorer, verify that mousing over Latest Runs data opens tooltip and that tooltip arrow is centered on the latest run status icon
3. Verify that status is reflected for prior runs in latest runs "dots"

### How has the user experience changed?
**Before** - tooltip arrow is _ever so slightly_ off center to the left
![before](https://user-images.githubusercontent.com/11482842/178294063-efe4301e-4fb7-4137-895c-566ae300a332.png)

**After** - tooltip arrow is perfect and amazing
![after](https://user-images.githubusercontent.com/11482842/178294080-9f69ae7c-259f-4eb4-b414-cc7b3f562121.png)


### PR Tasks
- [x] Have tests been added/updated? _Covered by Percy_
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
